### PR TITLE
Basic

### PR DESCRIPTION
**Pull request by saketkum11**
refactored the code:

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
-       <Route path="/login" element={<Login/>}></Route>
-       <Route path="/signup" element={<Signup/>}></Route>
-       <Route path="/forgot" element={<Forgot/>}></Route>
-       <Route path="/mock" element={<Mockman/>}></Route>
-     </Routes>
+    <div className='App'>
+      <Routes>
+        <Route path='/' element={<Home />}></Route>
+        <Route path='/product' element={<Product />}></Route>
+        <Route path='/wishlist' element={<Wishlist />}></Route>
```
Comment (narharikale) [inline src/App.js:18]: These should be private routes?

**Review by narharikale — APPROVED** [review]
liked the implementation of compose function

File: src/Pages/Auth/Login.jsx [hunk 2]
```diff
@@ -1,76 +1,95 @@
...
+          </div>
+          <form
+            onSubmit={(e) => {
+              e.preventDefault();
+              fetchLoginDetails(newClient.email, newClient.password);
+              navigate('/');
+            }}
+          >
+            <div className=' bg-black-0 flex flex-column  m-t-8 wt-100 '>
+              <label for='email' className='bg-white-0 '>
```
Comment (xsaurabhsuthar) [inline src/Pages/Auth/Login.jsx:29]: suggestion: you can make an input component and reuse it.

**Review by xsaurabhsuthar — APPROVED** [review]

**Review by JowelTisso — APPROVED** [review]
Looks good

File: src/Pages/Auth/Login.jsx (hunk 2 above)
Comment (saketkum11) [reply src/Pages/Auth/Login.jsx:29]: okk .
Component suggestion.

File: src/App.js (hunk 1 above)
Comment (saketkum11) [reply src/App.js:18]: For a time being I have left it. I will add route to every project.

**Pull request merged**